diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -1062,11 +1062,12 @@ namespace diff_drive_controller
     wheel_speed_limiter_ = wheel_speed_limiter;
   }
 
-  void DiffDriveController::wheelSpeedLimiter(double& left, double& right)
+  void DiffDriveController::wheelSpeedLimiter(double& left_command, double& right_command,
+                                              double left_desired, double right_desired)
   {
     if (wheel_speed_limiter_)
     {
-      wheel_speed_limiter_(left, right);
+      wheel_speed_limiter_(left_command, right_command, left_desired, right_desired);
     }
```

Comment:
Polymorphism really is the proper way to do this. Once Rocksteady is migrated, it will be possible get rid of the `wheel_speed_limiter_` member variable.